src/authenticated/DrawerContent.tsx

```diff
@@ -27,11 +25,21 @@ const styles = (theme: Theme) =>
     }
   });
 
+const DRAWER_PROJECTS = gql`
+  query projectsDrawer {
+    allProjects(orderBy: name_ASC) {
+      # TODO: Only show active projects
```

Comment:
Comments like this would be better left out of the code and instead made into an issue on Github.

Reply:
Added as a requirement to https://github.com/codefordenver/members/issues/326